Slight adjustments to alert level announcements to make them a bit less specific. (#12808)
BeeStation/BeeStation-Hornet: config/game_options.txt

ALERT_RED_DOWNTO There is still an immediate serious threat to the station. Security may have weapons unholstered at all times, random searches are allowed and advised.
ALERT_DELTA Destruction of the station is imminent. All hands are instructed to obey all instructions given by heads of staff. Any violations of these orders can be punished by death. This is not a drill.
ALERT_ENGINEERING The security alert is now Yellow(Engineering). A critical engineering issue is present on the station. Don protective gear and personal oxygen systems. Command authority is transferred to the Chief Engineer for the duration of this alert.
ALERT_MEDICAL The security alert is now Yellow(Medical). A C-B-R-N threat is present aboard the station. Don protective gear and personal oxygen systems. Command authority is transferred to the Chief Medical Officer for the duration of this alert.
ALERT_GAMMA Central Command has ordered "GAMMA" security level on the station. The station is now under direct central command control. All Central command staff are to be obeyed at all costs.

> ALERT_BLACK Central Command has ordered "BLACK" security level on the station. All hands are to prepare for hostile boarding. Any violations of orders from security personnel are punishable by death. This is not a drill.

Reviewer: I agree with changing the wording so it can be used with pirates, but also think it should be applied automatically during a pirate boarding. Are you willing to do that as part of this PR?

Author: hmmm i wouldn't be opposed to doing that but i also feel like it'd be taking some player agency away.

Right now you don't HAVE to go full war economy on their ass to repel pirates necessarily. Forcing black would take that choice away from them.

Reviewer: How do players choose to go to code black? I thought this was only escalated to through admin means or code.

Author: the idea is that they can always ask CC to up the code for them. If there's no CC they can just go code red.

Reviewer: I still think it should be the default then. 

Hostile boarding party feels like the kind of thing that should always trigger when applicable, even if pirates are a much lesser threat than nuclear operatives.

Author: It's less hostile boarding party and more full mobilisation.

I've made pirates auto call it for now, i'll make it manually callable later.

ALERT_LAMBDA Central Command has ordered "LAMBDA" security level on the station. Any violations of orders from Heads of Staff and security can be punished by death. All hands are advised to evacuate if possible.

> ALERT_EPSILON Central Command has ordered "EPSILON" security level on the station. All employee contracts are now terminated. Your station is designated as a lost asset.

Reviewer: ```suggestion
ALERT_EPSILON Central Command has ordered "EPSILON" security level on the station. Your station is designated as a lost asset. Consider your contracts with Nanotrasen terminated.
```

I like the vaguely threatening wording of this better, from the original message.
